roles/container-engine/docker/templates/daemon.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+    "data-root": "{{ docker_data_root }}",
+    "exec-opts": [
+        "native.cgroupdriver=cgroupfs"
+    ],
+    "iptables": "{{ docker_iptables | bool }}"
+    "log-driver": "json-file",
+    "log-opts": {
+        "max-size": "100m",
+    },
+    "storage-driver": "{{ docker_storage_driver }}",
```

Comment:
This changes the current behaviour that lets docker decides what storage driver to use. Is it on purpose?

Reply:
For Docker EE most likely recommend with overlay2: https://success.docker.com/article/compatibility-matrix

For Docker CE most likely also recommend with overlay2, too: https://docs.docker.com/storage/storagedriver/select-storage-driver/#docker-engine---community

For Kubernetes CRI installation, also use overlay2 as default: https://kubernetes.io/docs/setup/cri/#docker

I guess this should be good enough to make overlay2 as default for kubespray?

Reply:
Same here. I don't disagree but it changes the current behaviour.
I'd recommend providing a separate PR to enforce the use of overlay2